MonsterTrainModdingAPI/MonsterTrainModdingAPI.cs

```diff
@@ -8,7 +8,7 @@ namespace MonsterTrainModdingAPI
     /// <summary>
     /// The entry point for the API.
     /// </summary>
-    [BepInPlugin("api.modding.train.monster", "Monster Train Modding API", "0.0.6")]
+    [BepInPlugin("api.modding.train.monster", "Monster Train Modding API", "0.0.7")]
```

Comment:
Apologies for the change in philosophy, but I'd rather we didn't increase version numbers in feature branches anymore. I'd like to keep them to their own branches, since it's easier to find when scrolling through commit history, and also since I need to update documentation before preparing the release. In other words, when this gets merged in, there will still be at least two more commits before we actually release, so it doesn't make sense to increase the version number yet.